cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -148,9 +144,6 @@ data QueryInShelleyBasedEra era result where
      -- QueryPoolRanking
      --   :: QueryInShelleyBasedEra era RewardProvenance
 
-     QueryLedgerState
```

Comment:
Why are we removing the `QueryLedgerState`? Can't we leave it for now until you're ready to replace it?